gapic/src/main/com/google/gapid/perfetto/models/BatterySummaryTrack.java

```diff
@@ -139,26 +159,187 @@ private String counterSQL() {
     return data;
   }
 
+  public ListenableFuture<Values> getValue(long id) {
```

Comment:
Maybe I missed something, couldn't this be just a List<Value>?  Similar to getSices in your other PR.

Reply:
So far the Selection objects in Counter track, Memory track, Battery track are a little different from others, you can see that they don't differentiate single selection and multi selection: 
1. There's only one `Values` class, while in other panels there's `Slice` class and `Slices` class.
2. The information window beneath is the same for single selection or multi selection.
Since there's no single selection then there wouldn't be a List<SingleSelectionClass>.

Note: This pattern comes from the original design in Counter track. I guess it's done because counter data is much simpler than the slices data. I'm a supporter to apply this pattern to other tracks in the future, seeing single selection just a special case of mult selection.